stl/inc/vector

```diff
@@ -372,7 +372,10 @@ public:
     using reference       = value_type&;
     using const_reference = const value_type&;
 
-    _Vector_val() noexcept : _Myfirst(), _Mylast(), _Myend() {}
+    _Vector_val() noexcept : _Myfirst{}, _Mylast{}, _Myend{} {}

```

Comment:
Strictly speaking, brace-initialization isn't required to be valid for "fancy pointers" (types that meet the requirements to be used as allocator pointers). I'd prefer that we continue to uniformly initialize these members with parens. (Also on 378.)

Reply:
I pushed a change to both of these constructors to use parentheses.